src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "shutdown (Android SDK API)"
+type: apiDoc
+shortDescription: Shut down the agent during runtime
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: 'New Relic API for Android mobile app monitoring: shut down agent.'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down
+---
+
+## Syntax
+
+```
+NewRelic.shutdown()
+```
+
+Shut down the agent within the current application lifecycle during runtime
+
+## Requirements
+
+Agent version 6.10.0 or higher.
+
+## Description
+
+This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.
+
+
+
+## Examples
+
+### Shut down the Android agent
+
+Example of shutting down the agent:
+
+```
+NewRelic.shutdown()
+```
```

Comment:
@ywang-nr Can we build this example out more? Are there any scenarios where something would go in the brackets () of NewRelic.shutdown()?

Reply:
This is all the developers need for shutting down the agent, no params are needed